src/main/resources/reference.conf

```
@@ -1339,10 +1339,12 @@ resourceTypes = {
     ownerRoleName = "steward"
     roles = {
       steward = {
-        roleActions = ["delete", "read_dataset", "read_data", "edit_dataset", "manage_schema", "ingest_data",  "update_data", "soft_delete", "hard_delete", "create_datasnapshot",  "share_policy::steward", "share_policy::custodian", "share_policy::ingester", "share_policy::snapshot_creator", "read_policies", "link_snapshot", "unlink_snapshot", "list_snapshots", "update_passport_identifier", "view_journal", "lock_resource", "unlock_resource"]
+        roleActions = ["delete", "edit_dataset", "update_data", "create_datasnapshot",  "share_policy::steward", "update_passport_identifier", "view_journal"]
```

I believe `create_datasnapshot` is included as part of `custodian`
```suggestion
        roleActions = ["delete", "edit_dataset", "update_data",  "share_policy::steward", "update_passport_identifier", "view_journal"]
```

---

jinx!